src/main/java/appeng/util/ConfigInventory.java

```diff
@@ -207,8 +207,10 @@ public void setStack(int slot, @Nullable GenericStack stack) {
                 // force amount to 0 in types-only mode
                 stack = new GenericStack(stack.what(), 0);
             } else if (!typesOnly && stack.amount() <= 0) {
-                // in stack mode, amounts of 0 or less clear the slot
-                stack = null;
+                // in stack mode, amounts of 0 or less will:
+                // - filter the slot to a minimum of 1 of that stack, if empty
+                // - clear the slot, if not empty
+                stack = getStack(slot) == null ? new GenericStack(stack.what(), 1) : null;
```

Comment:
This looks problematic for config inventories in `STORAGE` mode (currently only the bottom slots of an interface) where it should materialize one source. Can you 1) check that it is indeed problematic for those, and 2) make this only apply for the `CONFIG_STACKS` mode?

Reply:
Point 2 can be done, point 1 seems to be redundant as empty container items like these are already filtered out by the `AEBaseMenu::handleEmptyHeldItem` method and as a result this current method is never actually called for the contained zero-amount stack.